src/router-setup.js

```diff
@@ -203,18 +196,45 @@ function getHotspotSsid() {
  * @param {string} ipaddr - IP address of AP
  * @returns {boolean} Boolean indicating success of the command.
  */
-function startAP(ipaddr) {
+function startCaptivePortal(ipaddr) {
+  DEBUG && console.log('router-setup: startCaptivePortal ipaddr:', ipaddr);
   const ssid = getHotspotSsid();
-  return platform.setWirelessMode(true, 'ap', {ssid, ipaddr});
+
+  // We need the LAN to be configured to have a static IP address,
+  // and run dnsmasq (DHCP server and DNS server).
+
+  const ifname = null;
+  const netmask = '255.255.255.0';
+
+  if (!platform.setLanMode('static', {ipaddr, ifname, netmask})) {
```

Comment:
Can you prefix all the logs in this function with `router-setup: ` for consistency?